config/uv/uv.html

```diff
@@ -16,31 +16,34 @@
                 margin: 0;
                 padding: 0;
                 overflow: hidden;
-            }
+              }

```

Comment:
Reading notes: lines 19 to 22 are the only "substantive" change.  The others for this file are removing non-meaningful whitespace.